lib/siwapp_web/views/layout_view.ex

```diff
@@ -4,4 +4,16 @@ defmodule SiwappWeb.LayoutView do
   # Phoenix LiveDashboard is available only in development by default,
   # so we instruct Elixir to not warn if the dashboard route is missing.
   @compile {:no_warn_undefined, {Routes, :live_dashboard_path, 2}}
+
+  def new_button(assigns) do
+    path = assigns.conn.request_path
```

Comment:
who assigned you this task?

Reply:
I understood that if we found anything improvable, we should do it. No worries, I will just PR in a new branch customer_list affecting.